src/main/windows/html_email.ts

```diff
@@ -218,6 +215,9 @@ export function openHtmlEmailWindow(
     window
   )
   window.setBrowserView(sandboxedView)
+  sandboxedView.webContents.setZoomFactor(
+    DesktopSettings.state.zoomFactor * (1.2 ^ window.webContents.getZoomLevel())
```

Comment:
```suggestion
    DesktopSettings.state.zoomFactor *
      Math.pow(1.2, window.webContents.getZoomLevel())
```

Though this does not appear to affect behavior as `window.webContents.getZoomLevel()` seems to be always 1, so the value of the expression is always just `DesktopSettings.state.zoomFactor` and then the resize handler runs and overrides it.